### PR TITLE
Update template for case where there are no other test failures

### PR DESCRIPTION
**Pull request by Prestaul**
- no need to show failed test list when there are none there
- fixed typo in the template `tho` -> `to`
- added `breakOnError` option for skeemas (slight speed up)

Again, I did not check in the updated reports or README. @Muscula, let me know if you'd like me to do this.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,7 +81,8 @@ testRunner([
 			return validator;
 		},
 		test: function (instance, json, schema) {
-			return instance.validate(json, schema).valid;
+			// third argument is `breakOnError` to halt validation at first error
+			return instance.validate(json, schema, true).valid;
```
Comment (Muscula) [inline index.js]: I do not want to start adding special settings for each validator. The tests should run it in the complete standard way, that most users would use if they are just picking up the validator and start using it.

Comment (Muscula) [inline index.js]: The reason for this is also that themis is using a whole host of special settings in their benchmarks, and testing all kinds of configurations with different stuff disabled. I don't want to venture down that slippery slope.

**Comment by Muscula** [comment]
Regardig your question about checking in the .md files. For now I would like to run the benchmarks, and generate the report. So not checking them in is good.

File: index.js (hunk 1 above)
Comment (Muscula) [inline index.js]: If you revert this one line, I'd be happy to merge this pull request.

**Comment by Muscula** [comment]
Ok, I added the template changes. If you want to include the break on error. I suggest you do a seperate PR for that and we can discuss that there. But as mentioned I would like to run all validators at their very basic default settings.

**Comment by Prestaul** [comment]
:+1: I understand. When I wrote skeemas, I wrote it assuming that there were two basic modes of operation when validating a schema: validation, or error collection.  Either you just need to know if an object is valid or you need to know how it is invalid. Most libraries only do one or the other and, honestly, I'm not sure which makes the most sense as the default mode of operation. Because you are using The Suite as your benchmark, it has almost no impact which mode I'm in. Nearly every test schema is tiny and offers a single failure anyway.

**Comment by Muscula** [comment]
You might be right. Though every use I've had for a schema validator, I've always reported or logged all the errors somewhere. But there are probably use-cases for true/fals only.
